package.json

```diff
@@ -65,6 +65,7 @@
     "prettier": "^1.19.1",
     "react": "^16.12.0",
     "react-dom": "^16.12.0",
+    "react-query": "^3.2.0-beta.23",
```

Comment:
not sure if it's a good idea to also add react-query to the dev-dependencies, but it gave me auto-complete when importing from it :)